test/src/test/java/com/rbmhtechnology/vind/test/ServerTest.java

```diff
@@ -444,25 +448,6 @@ public void testTermFacetAccess() {
         assertEquals(2, result.getFacetResults().getTermFacets().get(term).getValues().size());
     }
 
-    @Rule
```

Comment:
Why this is deleted?

Reply:
Double dependencies are supported, so there should be no error if one matching server exists